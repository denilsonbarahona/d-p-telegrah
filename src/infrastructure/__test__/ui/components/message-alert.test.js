import React from "react";
import { render } from "@testing-library/react";
import MessageAlert from "@UI-components/message-alert";

describe("MessageAlert", () => {
  it("rendering without crashing", () => {
    const { queryByRole } = render(<MessageAlert counter={8} />);
    const Alter = queryByRole("alert");
    expect(Alter).toBeInTheDocument();
  });

  it("to match snapshot", () => {
    const { container } = render(<MessageAlert counter={8} />);
    expect(container).toMatchSnapshot();
  });
});
