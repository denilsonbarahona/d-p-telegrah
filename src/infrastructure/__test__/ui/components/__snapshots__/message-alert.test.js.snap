// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessageAlert to match snapshot 1`] = `
<div>
  <div
    aria-label="8 messages without read"
    class="sc-bczRLJ ldMgCU"
    role="alert"
  >
    8
  </div>
</div>
`;
